### ISSUE TITLE
environment_setup.sh bumps I hit

### ISSUE THREAD

**Issue by jamidwyer** [thinkshout/ts_recipes#1]
Just noting a few lessons from my set up, for future documentation or error logging or troubleshooting.

First, I had already installed git. I think in the process of correcting that, I (unnecessarily) changed permissions and ownership on things.

One problem I had repeatedly was mysql not starting. This was ultimately fixed by removing an error log.

I'd forgotten to add the XCode command line tools on one round of installation and got an error that BZip was missing. We might add a reminder to install CLT there.

After several rounds of re-installs, I kept seeing that processes were already running. To fix that, we removed .plist files in the LaunchAgents folder:
homebrew.mxcl.dnsmasq.plist
homebrew.mxcl.httpd22.plist
homebrew.mxcl.mysql.plist

After all of that was fixed, the site still didn't load, and I noticed it was looking for httpd-vhosts.conf in the wrong location (for me, should have been /Users/jami/Sites/httpd-vhosts.conf). I corrected it in httpd.conf, started apache, and voila, web site.

(To view a site, add a folder (for example, testfolder) to /Users/[yourcomputername]/Sites/ with an index.php, then load testfolder.dev in your browser.)

**Comment by wxactly** [comment]
In order to get a Drupal site running, you need to modify `.htaccess` and set `RewriteBase /`

**Comment by jamidwyer** [comment]
That is some amazing debugging -- I hadn't even hit the error that fixed yet.

**Comment by jamidwyer** [comment]
Adding another note from a few months later. Every time PHP updates, I end up with a broken environment. I assumed this was due to permissions mistakes when I set up my machine, but another possible culprit is that homebrew suggests you run some other commands, which modifies the plist and ended for me with a memory leak and several instances of php-fpm running. 

These homebrew helpful tips should be ignored, so any way we can alert users to trust the original build would be good.
